proto/blueprint.proto

```diff
@@ -647,6 +647,11 @@ message Row {
      * when thrashers are displayed inside a fixed/thrasher container.
      */
     ROW_TYPE_WEB_CONTENT = 3;
+    /**
+     * New carousel style on some new collection types such as 
+     * small story carousel on tablet
+     */
+    ROW_TYPE_PROGRAMATIC_CAROUSEL = 4;
```

Comment:
I think there is a small typo here - I think it should be `ROW_TYPE_PROGRAMMATIC_CAROUSEL`.

Reply:
Thank you for spotting this!